### ISSUE TITLE
Unable to display loss on graph

### ISSUE THREAD

**Issue by ChangyuYan** [autonomio/talos#269]
The y-coordinate range for displaying losses during training can be too small sometimes, when the losses are big. Could you fix the display window range? 

Thanks!

**Comment by mikkokotila** [comment]
Are you referring to `talos.live()`?

**Comment by ChangyuYan** [comment]
![Screen Shot 2019-03-11 at 10 50 53 AM](https://user-images.githubusercontent.com/27938057/54145637-9c4a4780-43eb-11e9-8f6b-0c742bfae816.png)

It's the graph on the left in this picture.

**Comment by mikkokotila** [comment]
I understand. This is intentional. If we don't fix to scale of 0 to 1 then the scale is continuously changing and provides no meaningful way to compare results. Also, when you are not on the chart, you know something is totally wrong. The exception here is of course continuous prediction tasks, where a value is not ideally between 0 and 1 typically. So maybe we could have an option that allows switching to dynamic y-scale.

**Comment by reinert** [comment]
I'm running into the same issue. It would be nice if we could set the graph's scale. Any tips where I could change it by hand?

**Comment by mikkokotila** [comment]
I've fixed this now, but it will take probably a few weeks before the change will reflect in the underlying pypi package. In the meantime, if you uninstall kerasplotlib from your machine and then: 

`pip install git+https://github.com/autonomio/kerasplotlib`

...you should be able to pass an argument `x_max` to the callback and set your own limit. Note that I did not test this yet but it's a pretty simple change so should work. 

Closing here.
